indicators/migrations/0004_auto_20190201_1435.py

```diff
@@ -0,0 +1,70 @@
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='countryindicator',
+            name='IND',
+            field=models.PositiveIntegerField(validators=[django.core.validators.MinValueValidator(0), django.core.validators.MaxValueValidator(100)], verbose_name='IND'),
+        ),
+        migrations.RenameField('countryindicator', 'IND', 'ind'),
+        migrations.AlterField(
+            model_name='countryindicator',
+            name='IVR',
+            field=models.PositiveIntegerField(validators=[django.core.validators.MinValueValidator(0), django.core.validators.MaxValueValidator(100)], verbose_name='IVR'),
+        ),
+        migrations.RenameField('countryindicator', 'IVR', 'ivr'),
+        migrations.AlterField(
+            model_name='countryindicator',
+            name='LTO',
+            field=models.PositiveIntegerField(validators=[django.core.validators.MinValueValidator(0), django.core.validators.MaxValueValidator(100)], verbose_name='LTO'),
+        ),
+        migrations.RenameField('countryindicator', 'LTO', 'lto'),
+        migrations.AlterField(
+            model_name='countryindicator',
+            name='MAS',
+            field=models.PositiveIntegerField(validators=[django.core.validators.MinValueValidator(0), django.core.validators.MaxValueValidator(100)], verbose_name='MAS'),
+        ),
+        migrations.RenameField('countryindicator', 'MAS', 'mas'),
+        migrations.AlterField(
+            model_name='countryindicator',
+            name='PDI',
+            field=models.PositiveIntegerField(validators=[django.core.validators.MinValueValidator(0), django.core.validators.MaxValueValidator(100)], verbose_name='PDI'),
+        ),
+        migrations.RenameField('countryindicator', 'PDI', 'pdi'),
+        migrations.AlterField(
+            model_name='countryindicator',
+            name='UAI',
+            field=models.PositiveIntegerField(validators=[django.core.validators.MinValueValidator(0), django.core.validators.MaxValueValidator(100)], verbose_name='UAI'),
+        ),
+        migrations.RenameField('countryindicator', 'UAI', 'uai'),
+        migrations.AlterField(
+            model_name='countryindicator',
+            name='iso_code',
+            field=models.CharField(max_length=3, unique=True, validators=[django.core.validators.MinLengthValidator(3), django.core.validators.MaxLengthValidator(3)]),
+        ),
+        migrations.AlterField(
+            model_name='countryindicator',
+            name='name',
+            field=models.CharField(max_length=200, unique=True),
+        ),
+        migrations.AddField(
+            model_name='countryindicator',
+            name='id',
+            field=models.AutoField(auto_created=True, primary_key=True,
+                                   serialize=False, verbose_name='ID'),
+            preserve_default=False,
+            ),
+        migrations.RunPython(populate_ids_for_indicator),
```

Comment:
Well... I'm slightly confused because of this line. I'm just wondering if you'd be so kind to clarify a few questions to me? 
 - Why do you need to use this statement here?
 - Why did you decide to use `RunPython` insead of `RunSQL` which is much more efficient?
 - Why did you define `populate_ids_for_indicator` outside migration. Do you plan to use it more than once?